src/main/java/net/pms/dlna/LibMediaInfoParser.java

```diff
@@ -14,6 +14,7 @@
 import net.pms.util.ImagesUtil;
 import org.apache.commons.codec.binary.Base64;
 import org.apache.commons.imaging.ImageReadException;
+import org.apache.commons.lang3.StringUtils;
```

Comment:
@Nadahar Sorry, i don't understand why it need to have 2 import for the same class.
Are they all really necessary ?